HW_12/hw_12.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types=1);
+
+error_reporting(E_ALL);
+ini_set('display_errors', '1');
+
+//NORMAL level
+//Task 1
+
+function getStr()
+{
+    echo "Hello, World!" . "<br>";
+}
+
+getStr();
+
+//Task 2
+function getLeapYear(int $year): bool
+{
+    return !($year % 4 <> 0) && (($year % 100 <> 0 || $year % 400 == 0));
+}
+
+var_dump(getLeapYear(2001));
+echo "<br>";
+
+//Task 3
+function getAbr(string $str): string
+{
+    $res = '';
+    $temp = explode(' ', $str);
+    foreach ($temp as $key => $item) {
+        $res .= mb_strtoupper(mb_substr($item, 0, 1));
+    }
+    return $res;
+}
+
+echo getAbr('привет мир'), "<br>";
+echo getAbr('hello world'), "<br>";
+
+//Task 4
+function revers(string $str): string
+{
+    $res = '';
+    for ($i = mb_strlen($str); $i >= 0; $i--) {
+        $res .= mb_substr($str, $i, 1);
+    }
+    return $res;
+}
+
+echo revers('привет word @@@@'), "<br>";
+
+//Task 5
+function isSimple(int $x): bool
+{
+    if ($x === 1) {
+        return false;
+    }
+    for ($i = 2; $i <= sqrt($x); $i++) {
+        if ($x % $i === 0) {
+            return false;
+        }
+    }
+    return true;
+}
+
+var_dump(isSimple(5));
+echo "<br>";
+
+//Task 6
+function getLen($str): int
+{
+    $temp = explode(' ', $str);
+    $lastWord = array_pop($temp);
+    return strlen($lastWord);
+}
+
+echo getLen('hello, wordl !'), "<br>";
```

Comment:
хитришь) знаки препинания сразу после слова идут
```suggestion
echo getLen('hello, wordl!'), "<br>";
```